### PR TITLE
🌶️ spicy post alert

### PR DESCRIPTION
**Pull request by mbifulco**

### CODE REVIEW

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+Throughout the course of the experiment, the experiment was conducted on 40 volunteers; all men between the ages of 20 and 50, with a variety of education and economic backgrounds.
+
+The results of the experiment were extremely unexpected.
+
+### The Experiment: Setup
+
+Participants who were asked to take part in a study on learning and memory. Each study was led guided by an _Experimenter_ dressed in a lab coat, who explained to participants that they'd be doing a "scientific study on memory and learning". When they arrived, participants were paired with an actor who they were _told_ was another participant in the study.
+
+The actor was actually a confederate of the researcher, and was instructed to play the role of a _Learner_.
```
Comment (chandler-berty) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:22]: Tell me you live in NC without telling me you live in NC.

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+<br />
+
+However, the actual results of the experiment revealed a surprising truth about human behavior:
+
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/actual-results"
+  alt="Actual Results: 59.1% completed the full series of shocks, 40.9% refused to continue at some point"
+  caption="The experimental results were much more grim."
+/>
+
```
Comment (chandler-berty) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:87]: But, that study was in 1961. We've evolved culturally since then, I don't think this could happen now? 

Nope, replicated in 2009, we're still awful: https://www.apa.org/news/press/releases/2008/12/replicating-milgram

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+  publicId="posts/twitter-and-the-perils-of-obedience/experiment-3"
+  alt=""
+/>
+
+<br />
+<Colophon />
+
+## Footnote: on the ethics of the Milgram Experiment
+
+It's important to note that in the years since Milgram Experiment was conducted, it has been criticized for its methodology. [Research Ethics](https://en.wikipedia.org/wiki/Research#Research_ethics) is a complex and nuanced topic, and I don't believe that the experiment would be approved by modern ethics boards in its original form. However, I believe that the results of the experiment are still relevant, and that the experiment itself is still a valuable tool for understanding human behavior.
```
Comment (chandler-berty) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:150]: The later study made some modifications to get by ethics boards. Like lowering the level of the simulated shock and sceening candidates to find those who might be adversely affected.

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+Since then, another hasty poll has resulted in an absolutely staggering policy change: all\* accounts that were previously suspended for violating Twitter's rules will be reinstated.
+
+<Tweet align="center" tweetLink="elonmusk/status/1595473875847942146" />
+
+Pandora's box has been opened, and the monsters are coming back. Twitter is now a platform for hate speech, misinformation, and disinformation. It's will become a cesspool of conspiracy theories, and a breeding ground for countless nightmares and real-world dangers.
+
+This is a very real, very observable truth: countless racists, abusers, and other dangerous people are being given "amnesty" to come back the platform.
+
+I've seen many others abandon their twitter accounts in the wake of these changes, but -- friends -- I'm honestly shocked that so many people choose to remain.
```
Comment (chandler-berty) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:112]: I don't disagree with you here, but I think if you soften your tone it would make your point stronger. This is also a very soft suggestion. I think the way you have it is fine. 

Current, "I am shocked there are so many here. What is wrong with you all? 

Possible future, "There are still a ton there. That surprises me, and I'm trying to make sense of it. Part of the reason could be Milgram's obedience.. Part is clearly revenue (which you mention). Part is switching costs. Part is the "bystander affect". 

You could even have a call to action in this.. asking people to tell you why they aren't leaving. Then do a follow up stating why. 

There is a really cool article on /r/nootropics where a guy took a poll then posted the info: https://troof.blog/posts/nootropics/

000656@2x

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+Why are so many people still on twitter, administer electric shocks to the collective public by keeping their accounts alive?
+
+I'm reminded by the words of Stanley Milgram, in his reflection on the results of his now-infamous experiment:
```
Comment (chandler-berty) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:116]: Stanford prison experiments also come to mind. Where participants were so consumed with their assumed role they lost morality.

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+Under Musk's leadership, Twitter's employee base has been slashed from 7,500 to approximately 2,000 in the last month, in an attempt to cut costs after a purchase that only made sense for a rich, vein billionaire who wanted to flex his ego.
+
+I've seen plenty of discussion online that many of twitter's remaining engineers are likely tied to H1B visas, and are unable to leave their job without risking deportation. There is no doubt another contingent of employees who believe in Musk's vision, who are bought into the idea of being an elite engineer working on a desperately slim team, and are willing to work for him despite the fact that he's a complete and utter asshole.
+
+I don't think it is a stretch to say that employees at Twitter are being treated poorly, and have been given increasingly shocking orders from their new CEO (for example, [1](https://twitter.com/ZoeSchiffer/status/1593649356661436417), [2](https://twitter.com/alexeheath/status/1595701492006014977?t=7MMiqyX0-LWIXExz6QtkAQ&s=19)).
```
Comment (chandler-berty) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:96]: I think it would be interesting if you drew this analogy to a non-tech product. 

Would you go to your local coffee shop if you knew somebody just bought it and was making all of your favorite baristas sign a form to go into "hardcore" mode? 

Would you buy product made if you knew it's processes abused employees? We all do this, every day.

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+I've seen many others abandon their twitter accounts in the wake of these changes, but -- friends -- I'm honestly shocked that so many people choose to remain.
+
+Why are so many people still on twitter, administer electric shocks to the collective public by keeping their accounts alive?
+
+I'm reminded by the words of Stanley Milgram, in his reflection on the results of his now-infamous experiment:
+
+> Ordinary people, simply doing their jobs, and without any particular hostility on their part, can become agents in a terrible destructive process. Moreover, even when the destructive effects of their work become patently clear, and they are asked to carry out actions incompatible with fundamental standards of morality, relatively **few people have the resources needed to resist authority.**
+
+For many people, Twitter was a place to connect with friends, family, and colleagues. I know many people with follower counts in the tens of thousands, who have built a career around their twitter presence. I can understand a deep hesitancy to abandon a platform that has been so central to your life.
```
Comment (chandler-berty) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:120]: https://en.wikipedia.org/wiki/Psychological_inertia

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+
+But fam, seriously -- **if not now, when?**
+
+## A call to action
+
+Friends, it's time to leave Twitter. Shutter your accounts, delete the app, and move on. I know it's hard, but it's time. Every moment you spent on Twitter is feeding legitimacy to a platform that is enabling abusers, endangering global politics, and destroying the fabric of our society.
+
+It is no longer acceptable to stay aboard the ship as it sinks - to stay active on Twitter despite their new policies and leadership is a vanity, a bath in your own privilege.
+
+Bring your voice and your conversations elsewhere, and send the message that with its current leadership, Twitter is no longer a place for you.
```
Comment (chandler-berty) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:130]: I'm also concerned about the precedent this sets if Musk succeeds. Will every tech company be bought, gutted, employees thrown around like trash?

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+date: 11-28-22
+published: true
+path: twitter-and-the-perils-of-obedience
+---
+
+## Milgram's Experiment
+
+In 1961, a Yale University researcher named Stanley Milgram conducted a series of experiments as a part of his research on behavioral psychology. The goal of the experiment was to test participants' tendency to obey authority figures when under pressure.
+
+Throughout the course of the experiment, the experiment was conducted on 40 volunteers; all men between the ages of 20 and 50, with a variety of education and economic backgrounds.
```
Comment (tpgriffin89) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:14]: Confusing ... would reword to "The experiment was conducted on 40 volunteers; all men between the ages of 20 and 50, and each with a variety of education and economic backgrounds."

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+
+Milgram later summarized findings from the experiment in an essay titled "The Perils of Obedience". The results were extremely unexpected.
```
Comment (tpgriffin89) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:16]: Period before closing a quote, so would read "The Perils of Obedience."

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+
+### The Experiment: Setup
+
+Participants who were asked to take part in a study on learning and memory. Each study was led guided by an _Experimenter_ dressed in a lab coat, who explained to participants that they'd be doing a "scientific study on memory and learning". When they arrived, participants were paired with an actor who they were _told_ was another participant in the study.
```
Comment (tpgriffin89) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:20]: Same as before with the closing quote.

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+
+The actor was secretly working with the researcher, and during the experiment was instructed to play the role of a _Learner_.
+
+The study participant was given the role of _Teacher_. The experimenter then explained that the study would involve the Teacher asking the Learner series of questions. For each incorrect answer, the Teacher would be asked to hit a button to administer an electric shock to the actor. Subsequent incorrect answers would result in a shock of increasing intensity, from 15V to 450V.
```
Comment (tpgriffin89) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:24]: Not immediate clear who the participant was, maybe "The volunteer study participant" or something similar?

Comment (tpgriffin89) [reply src/data/posts/twitter-and-the-perils-of-obedience.mdx:24]: .... that the study would involve the Teacher asking the Learner **a** series of questions.

Comment (tpgriffin89) [reply src/data/posts/twitter-and-the-perils-of-obedience.mdx:24]: For each incorrect answer, the Teacher would be asked to hit a button <s>to</s> **that would** administer an electric shock to the actor. Subsequent incorrect answers would result in a shock of increasing intensity, from 15V to 450V.

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 13]
```diff
@@ -0,0 +1,172 @@
...
+
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/experiment-photos"
+  alt="Photos from the original Milgram experiment, credit Stanley Milgram."
+  caption="Photos from the original Milgram experiment, credit Stanley Milgram"
+/>
+
+### What would you do?
+
+Let's take a minute to pause here - put yourself in the shoes of the teacher. You're in a lab, and you're told that you're taking part in a study on learning and memory. Everything you're seeing and hearing seems to be controlled, official, and legititmate.
```
Comment (tpgriffin89) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:57]: Teacher capitalized to be consistent.

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 14]
```diff
@@ -0,0 +1,172 @@
...
+
+Do you think you would continue to administer shocks to the learner, even when they were clearly in pain?
```
Comment (tpgriffin89) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:59]: Same as above, Learner capitalized.

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 15]
```diff
@@ -0,0 +1,172 @@
...
+
+Recreations of this experiment have produced similar results, [as recently as 2009](https://www.apa.org/news/press/releases/2008/12/replicating-milgram).
+
+## Obedience and the Perils of Twitter
+
+I can't help but draw a connection between the Milgram experiment and the current state of Twitter. The experiment was designed to test the willingness of people to obey authority figures. Twitter's recent acquisition by a megalomaniacal celebrity narcissist is an amplified example of this phenomenon in action.
+
+Under Musk's leadership, Twitter's employee base has been slashed from 7,500 to approximately 2,000 in the last month, in an attempt to cut costs after a purchase that only made sense for a rich, vein billionaire who wanted to flex his ego.
+
+I've seen plenty of discussion online that many of twitter's remaining engineers are likely tied to H1B visas, and are unable to leave their job without risking deportation. There is no doubt another contingent of employees who believe in Musk's vision, who are bought into the idea of being an elite engineer working on a desperately slim team, and are willing to work for him despite the fact that he's a complete and utter asshole.
```
Comment (tpgriffin89) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:96]: The H1B piece of this has been largely disputed fwiw.

Instead of "who are bought into" would change to just "are bought into"

The "complete and utter asshole" line is pure opinion and may discredit the argument you're trying to make. Would maybe tie it to his "complete disregard for moderation best practices and repeatedly lying about his account review policies."

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 16]
```diff
@@ -0,0 +1,172 @@
...
+
+I don't think it is a stretch to say that employees at Twitter are being treated poorly, and have been given increasingly shocking orders from their new CEO (for example, [1](https://twitter.com/ZoeSchiffer/status/1593649356661436417), [2](https://twitter.com/alexeheath/status/1595701492006014977?t=7MMiqyX0-LWIXExz6QtkAQ&s=19)).
```
Comment (tpgriffin89) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:98]: Again, purely an opinion here ... he's demading a lot of the team yes, but that doesn't mean they're being treated poorly. Especially in relation to a lot of jobs in America, it's still a champagne problem at best.

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 17]
```diff
@@ -0,0 +1,172 @@
...
+
+Since rumors of Musk's takeover, legions of advertisers have pulled their ad spend from Twitter, for fear of being associated with the company. The company's income has plummeted, and Musk is clearly trying to cut costs wherever he can, while adding features as a part of a desperate money grab to keep the company from going bankrupt.
```
Comment (tpgriffin89) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:100]: Not sure this hits the point you want it to, of course he's desperate to generate revenue, he just spent $44B on something that's losing a lot of money. The real message here is that advertisers are leaving ... why? Because it isn't clear what direction the platform will go, and as the company endures severe turnover, it's not worth the spend in the near term. There are some great threads from actual advertisers out there right now, will try to find links and text you them!

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 18]
```diff
@@ -0,0 +1,172 @@
...
+
+I left twitter recently, shortly after a hasty user poll resulted in the reinstatement of former (and twice-impeached) president Trump's account.
+
+<Tweet align="center" tweetLink="irreverentmike/status/1594455660040572928" />
+
+Since then, another hasty poll has resulted in an absolutely staggering policy change: all\* accounts that were previously suspended for violating Twitter's rules will be reinstated.
+
+<Tweet align="center" tweetLink="elonmusk/status/1595473875847942146" />
+
+Pandora's box has been opened, and the monsters are coming back. Twitter is now a platform for hate speech, misinformation, and disinformation. It's will become a cesspool of conspiracy theories, and a breeding ground for countless nightmares and real-world dangers.
```
Comment (tpgriffin89) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:110]: Devil's advocate viewpoint ... it's always been a platform for these things, but you could argue Reddit is the same thing. It's only a platform for these things, if you're seeking it out as a user. I see very little of this, and 99% of the time I only see it when someone is denouncing it. It's a perpetual echo chamber if anything, and allows people to feel that their horrible beliefs are shared with millions of people, even if the reality is a small fraction of that.

Comment (tpgriffin89) [reply src/data/posts/twitter-and-the-perils-of-obedience.mdx:110]: This is a great place to tie in the insurrection though ... the real-world danger is most pertinent to the argument, and was a very tangible event that took place in large part because of the vitriol spewed by the now unblocked orange man.

File: src/data/posts/twitter-and-the-perils-of-obedience.mdx [hunk 19]
```diff
@@ -0,0 +1,172 @@
...
+
+This is a very real, very observable truth: countless racists, abusers, and other dangerous people are being given "amnesty" to come back the platform.
+
+I've seen many others abandon their twitter accounts in the wake of these changes, but -- friends -- I'm honestly shocked that so many people choose to remain.
```
Comment (tpgriffin89) [inline src/data/posts/twitter-and-the-perils-of-obedience.mdx:114]: Would cite my reasons from my text this AM ... it's still an incredible new aggregator, a place to learn about other people, technology, ideas, etc. Yes it's filled to the brim with a lot of terrible people, but I've personally noticed very little change in my own feed, which has consistently been filled with tech founders and VCs for many years, and continues that way today. I stay, because until another platform is able to attract and generate that content at scale, I will feel uninformed as a human. It's my news source, and I'm smart enoguh to be able to sift through the data that's presented to me, in order to identify real news from fake ... unfortunately, not everyone can do that lol.
